runtime/parachains/src/configuration/migration/v5.rs

```diff
@@ -352,13 +361,13 @@ executor_params                          : Default::default(),
 		}
 	};
 
-	let v4 = V4ActiveConfig::<T>::get()
+	let v4 = v4::ActiveConfig::<T>::get()
 		.defensive_proof("Could not decode old config")
 		.unwrap_or_default();
```

Comment:
The `unwrap_or_default` did not help any, since the following `V5ActiveConfig::<T>::set` ignore it.